README.md

```diff
@@ -67,6 +67,7 @@ More information about these configurations can be found in the [Deploy Sematic
 | `aws.storage.bucket`                                      | AWS S3 bucket name for cloud artifacts                        | `my-s3-bucket`                          |
 | `secret.create`                                           | Creates a K8S secret for Postgres URL                         | `true`                                  |
 | `database.url`                                            | Fully qualified Postgres database URL                         | `postgres://postgresql:5432/postgres`   |
+| `deployment.worker_count`                                 | Number of WSGI workers per Sematic server pod (API only)      | `1`                                     |
```

Comment:
Please mention that the `socketio.dedicated` configuration must also be enabled if increasing this.